pkg/utils/logger/middleware.go

```diff
@@ -26,6 +39,14 @@ func GinLogger(server string) gin.HandlerFunc {
 		// Handle the request
 		c.Next()
 
+		// TODO prometheus metrics - decrement active requests
+		// switch server {
+		// case "quarterdeck":
+		// 	metrics.ActiveQDRequests.Dec()
+		// case "tenant":
+		// 	metrics.ActiveTenantRequests.Dec()
+		// }
```

Comment:
Maybe this should go all the way to the bottom of the func so that active requests include logging?